FeedSX/navigation/SwitchComponent.tsx

```diff
@@ -13,9 +13,10 @@ const SwitchComponent = () => {
   return (
     <>
       <NavigationContainer ref={navigationRef}>
-        <Stack.Navigator>
+        <Stack.Navigator screenOptions={{headerShown:false}}>
           <Stack.Screen name={'UniversalFeed'} component={UniversalFeed} />
```

Comment:
Either make emuns or constants in a separate file and use it here.

Reply:
resolved